less/sizing.less

```diff
@@ -0,0 +1,22 @@
+
+.layout-small {
+    font-size: 12px;
+
+    .Checkbox {
```

Comment:
Right on, totally forgot about that syntax. You like the convention of `layout-small|medium|large` then I take it?